arbnode/dataposter/externalsignertest/externalsignertest.go

```diff
@@ -94,8 +98,14 @@ func NewServer(t *testing.T) (*http.Server, *SignerAPI) {
 	pool := x509.NewCertPool()
 	pool.AppendCertsFromPEM(clientCert)
 
+	ln, err := net.Listen("tcp", "localhost:0")
```

Comment:
```suggestion
	ln, err := net.Listen("tcp", "127.0.0.1:0")
```

Beware that using localhost might resolve either to IPV4 or to IPV6. It is best to be explicit and use the IPV4 address. I think the CI tests are failing because of this.